pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -407,6 +407,9 @@ func Generate(cfg binding.Config) error {
 			} else if standard.ComplyABI(cfg.Manifest, standard.Nep11NonDivisible) == nil {
 				mfst.ABI.Methods = dropStdMethods(mfst.ABI.Methods, standard.Nep11NonDivisible)
 				ctr.IsNep11ND = true
+			} else if standard.ComplyABI(cfg.Manifest, standard.Nep11WithRoyalty) == nil {
+				mfst.ABI.Methods = dropStdMethods(mfst.ABI.Methods, standard.Nep11WithRoyalty)
+				ctr.IsNep11D = true
```

Comment:
This code is unreachable, because NEP11 standard is checked firstly. You need a separate `if std == manifest.NEP24StandardName` check to be added in a separate clause.